packages/protocol-kit/src/adapters/ethers/contracts/contractInstancesEthers.ts

```diff
@@ -44,25 +43,40 @@ import SignMessageLibContract_V1_3_0_Ethers from './SignMessageLib/v1.3.0/SignMe
 import SignMessageLibContract_V1_4_1_Ethers from './SignMessageLib/v1.4.1/SignMessageLibContract_V1_4_1_Ethers'
 import SimulateTxAccessorContract_V1_3_0_Ethers from './SimulateTxAccessor/v1.3.0/SimulateTxAccessorContract_V1_3_0_Ethers'
 import SimulateTxAccessorContract_V1_4_1_Ethers from './SimulateTxAccessor/v1.4.1/SimulateTxAccessorContract_V1_4_1_Ethers'
+import SafeContract_v1_3_0_Ethers from '@safe-global/protocol-kit/adapters/ethers/contracts/Safe/v1.3.0/SafeContract_v1_3_0_Ethers'
+import EthersAdapter from '../EthersAdapter'
+import { SafeTransactionData } from 'packages/safe-core-sdk-types/dist/src'
```

Comment:
I think we can import it from '@safe-global/safe-core-sdk-types' (which we already have in line 24), right?

Reply:
done!